# HG changeset patch
# User Sergey Mironov <grrwlf@gmail.com>
# Date 1404712998 -14400
#      Mon Jul 07 10:03:18 2014 +0400
# Node ID 9907088f9fa81773e124a551b1d99cdc7d11f9d1
# Parent  cf453f48d28b66c2f8f591410c3512a90107fe48
Add basic support for 'meta' tag

diff --git a/lib/ur/basis.urs b/lib/ur/basis.urs
--- a/lib/ur/basis.urs
+++ b/lib/ur/basis.urs
@@ -806,6 +806,7 @@
 val head : unit -> tag [Data = data_attr] html head [] []
 val title : unit -> tag [Data = data_attr] head [] [] []
 val link : unit -> tag [Data = data_attr, Id = id, Rel = string, Typ = string, Href = url, Media = string] head [] [] []
+val meta : unit -> tag [Nam = string, Content = string] head [] [] []
 
 val body : unit -> tag [Data = data_attr, Onload = transaction unit, Onresize = transaction unit, Onunload = transaction unit, Onhashchange = transaction unit]
                        html body [] []
